apps/src/music/views/MusicPlayView.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './music-play-view.module.scss';
+import Link from '@cdo/apps/componentLibrary/link';
+
+const codeOrgLogo = require(`@cdo/static/music/code-dot-org-white-logo.png`);
+
+const MusicPlayView: React.FunctionComponent = () => {
+  return (
+    <div className={moduleStyles.musicPlayViewContainer}>
+      <div className={moduleStyles.musicPlayViewCard}>
+        <img src={codeOrgLogo} alt="Code.org" />
+        <Heading2>My Awesome Mix</Heading2>
+        <BodyTwoText>
+          made with Music Lab on{' '}
```

Comment:
Is this space just a prettier artifact?

Reply:
This one is intended since we need a space between text and the link